docs/Short Instructions.md

```diff
@@ -0,0 +1,320 @@
+# How to add new backend in occa-transpiler 
```

Comment:
Please rename doc file to "howto_add_new_accelerator.md".
The good practice is to use camel case or snake case for file name and avoid white spaces.

Reply:
Done.